src/views/ChallengeList.vue

```diff
@@ -0,0 +1,132 @@
+<template>
+  <v-row v-if="!loading">
+    <v-col>
+      <v-chip label variant="text" prepend-icon="mdi-tag-outline">
+        {{ $t('Challenge.ChallengeCount', { count: challengeTotal }) }}
+      </v-chip>
+      <LoadedCountChip :loadedCount="challengeList.length" :totalCount="challengeTotal" />
+      <FilterMenu kind="challenge" :currentFilter="currentFilter" :currentType="currentType" @update:currentFilter="toggleChallengeFilter($event)" @update:currentType="toggleChallengeType($event)" />
+    </v-col>
+  </v-row>
+
+  <v-row class="mt-0">
+    <v-col v-for="challenge in challengeList" :key="challenge" cols="12" sm="6" md="4" xl="3">
+      <v-card :id="'challenge_' + challenge.id" :class="'border-transparent'" @click="goToChallenge(challenge)">
```

Comment:
could be moved to a dedicated component (future PR)

Reply:
also, mouse hover doesn't show it's a url (but this is recurring problem due to the usage of @click instead of <a> or router-link, we'll need to find a better way for all of these occurences)